src/test/java/com/bookshop/repository/BookRepositoryTests.java

```diff
@@ -20,26 +21,95 @@ public class BookRepositoryTests {
     @Autowired
     private BookRepository bookRepository;
 
-    //@Test
-    @DisplayName("Find only book with category 1")
-    @Sql(scripts = "classpath:database/books/add-one-book-with-first-category.sql",
+    @Test
+    @DisplayName("Find only book with correct category 1")
+    @Sql(scripts = {"classpath:database/books/remove-all-books-and-categories.sql",
```

Comment:
consider keeping repetitive strings in constants. it would be very helpful if you decide to rename/move the script file